platforms/platform-facebookmessenger/README.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Facebook Messenger Platform Integration'
+excerpt: 'The Facebook Messenger platform integration allows you to build custom Messenger Bots using Jovo.'
+---
+
+# Facebook Messenger Platform Integration
+
+The Facebook Messenger [platform integration](https://v4.jovo.tech/docs/platforms) allows you to build custom Messenger bots using Jovo.
+
+
+## Getting Started
+
+You can install the plugin like this:
+
+```sh
+$ npm install @jovotech/platform-facebookmessenger
+```
+
+Add it as plugin to your [app configuration](https://v4.jovo.tech/docs/app-config), e.g. `app.ts`:
+
+```typescript
+import { App } from '@jovotech/framework';
+import { FacebookMessengerPlatform } from '@jovotech/platform-facebookmessenger';
+// ...
+
+const app = new App({
+  plugins: [
+    new FacebookMessengerPlatform(),
+    // ...
+  ],
+});
+```
+
+## Configuration
+
+You can configure the Facebook Messenger platform in the [app configuration](https://v4.jovo.tech/docs/app-config), for example `app.ts`:
+
+```typescript
+import { FacebookMessengerPlatform } from '@jovotech/platform-facebookmessenger';
+
+// ...
+
+const app = new App({
```

Comment:
The example is incorrect, `FacebookMessengerPlatform` is missing and the session-property should be inside of the configuration of `FacebookMessengerPlatform` then, but this was probably just forgotten.